### PR TITLE
change create performance evaluation screen layout

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Create Performance Evaluation Screen (Step 1) #5](https://github.com/nerubia/kh360-react/issues/5#issuecomment-1793951633)

### CODE REVIEW

File: src/features/admin/evaluation-administrations/create/CreateEvaluationHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const CreateEvaluationHeader = () => {
+  return (
+    <h1 className='text-2xl text-primary-600 font-bold'>Create Evaluation</h1>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/create/CreateEvaluationHeader.tsx:3]: text-primary-500 is the default color

File: src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx [hunk 2]
```diff
@@ -90,83 +90,84 @@ export const CreateEvaluationForm = () => {
...
-            <h1 className='text-lg font-bold'>Evaluation Name</h1>
-            <Input
-              label='Evaluation name'
-              name='name'
-              placeholder='Evaluation name'
-              value={formData.name}
-              onChange={handleInputChange}
-              error={validationErrors.name}
-            />
+      <div className='flex flex-col w-1/2 gap-4'>
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx:93]: should be `md:w-1/2` 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx (hunk 2 above)
Comment (emmaacha) [reply src/features/admin/evaluation-administrations/create/CreateEvaluationForm.tsx]: nice catch! 🙌

**Review by jatnerubia — APPROVED** [review]
Cool

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**